model/trace/messaging.yaml

```diff
@@ -170,3 +170,11 @@ groups:
         tag: tech-specific-rocketmq
       - ref: messaging.rocketmq.consumption_model
         tag: tech-specific-rocketmq
+  - id: messaging.gcp_pubsub
+    type: attribute_group
+    extends: messaging
+    brief: >
+      Attributes for Google Cloud Pub/Sub
+    attributes:
+      - ref: messaging.gcp_pubsub.message.ordering_key
```

Comment:
is the requirement level intentionally `Recommended`? 

Just wondering because of the brief:
> If the attribute is not present, the message does not have an ordering key.

It sounds like conditionally required, because absence of that attribute has implicit meaning.

Reply:
No not intentional, good catch.

I'm not familiar with semantic conventions, yet. Updated it.